### PR TITLE
Add flag to indicate when focusable is true or not | Fixes #536

### LINKED ISSUE #536: IE: Tabbing through the document focuses SVG elements
Clay SVG's need the attribute `focusable="false"`

### PR DESCRIPTION
**Pull request by matuzalemsteles**

### CODE REVIEW

File: packages/clay-icon/src/ClayIcon.js [hunk 1]
```diff
@@ -25,6 +25,15 @@ ClayIcon.STATE = {
 	 */
 	elementClasses: Config.string(),
 
+	/**
+	 * Flag to indicate when svg focusable or not.
```
Comment (carloslancha) [inline packages/clay-icon/src/ClayIcon.js]: Typo: Flag to indicate if the svg is focusable or not

File: packages/clay-icon/src/ClayIcon.soy [hunk 2]
```diff
@@ -20,6 +21,12 @@
 			{/if}
 		"
 
+		{if $focusable}
```
Comment (carloslancha) [inline packages/clay-icon/src/ClayIcon.soy]: Can this be written `focusable="{$focusable == true}"` ?

File: packages/clay-icon/src/ClayIcon.js (hunk 1 above)
Comment (matuzalemsteles) [reply packages/clay-icon/src/ClayIcon.js]: That looks better, thanks!

File: packages/clay-icon/src/ClayIcon.soy (hunk 2 above)
Comment (matuzalemsteles) [reply packages/clay-icon/src/ClayIcon.soy]: No, the attributes do not allow **bool** to be used ☹️. To get as close to that `focusable="{$focusable == true ? 'true' : 'false'}"`(is that what you meant?). Sounds good to you?

Comment (carloslancha) [reply packages/clay-icon/src/ClayIcon.soy]: Nah, leave it as it was 😄 Thx!

**Comment by carloslancha** [comment]
Merged, thx!

**Pull request merged**